tests/unit/postTestBuildComment.ts

```diff
@@ -49,15 +54,18 @@ const message = `:test_tube::test_tube: Use the links below to test this adhoc b
 :eyes: [View the workflow run that generated this build](https://github.com/Expensify/App/actions/runs/1234) :eyes:
 `;
 
+const asMutable = <T>(value: T): Writable<T> => value as Writable<T>;
+
 describe('Post test build comments action tests', () => {
     beforeAll(() => {
         // Mock core module
-        core.getInput = mockGetInput;
-        GithubUtils.createComment = mockCreateComment;
+        asMutable(core).getInput = mockGetInput;
     });
 
     test('Test GH action', async () => {
-        when(core.getInput).calledWith('PR_NUMBER', {required: true}).mockReturnValue(12);
+        when(core.getInput)
+            .calledWith('PR_NUMBER', {required: true})
+            .mockReturnValue(12 as unknown as string);
```

Comment:
why number is casted to string?

Reply:
This is casted to string to comply with TS as the return type of code.getInput is a string and if we just change 12 to string the test is not passing, so this was the best solution I could find without changing the logic.